src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -30,86 +30,98 @@
-        Student studentToDelete = model.getFilteredStudentList().get(INDEX_FIRST_PERSON.getZeroBased());
-
         DeleteCommand deleteCommandTutor = new DeleteCommand(INDEX_FIRST_PERSON, PersonType.TUTOR);
-        DeleteCommand deleteCommandStudent = new DeleteCommand(INDEX_FIRST_PERSON, PersonType.STUDENT);
-
         String expectedMessageTutor = String.format(DeleteCommand.MESSAGE_DELETE_TUTOR_SUCCESS, tutorToDelete);
-        String expectedMessageStudent = String.format(DeleteCommand.MESSAGE_DELETE_STUDENT_SUCCESS, studentToDelete);
-
         ModelManager expectedModelTutor = new ModelManager(model.getAddressBook(), new UserPrefs());
-        ModelManager expectedModelStudent = new ModelManager(model.getAddressBook(), new UserPrefs());
-
         expectedModelTutor.deleteTutor(tutorToDelete);
-        expectedModelStudent.deleteStudent(studentToDelete);
 
-        // Successfully deleting a tutor for valid index for an unfiltered list
         assertCommandSuccess(deleteCommandTutor, model, expectedMessageTutor, expectedModelTutor);
-        // Successfully deleting a student for valid index for an unfiltered list
+    }
+
+    @Test
+    public void execute_validIndexUnfilteredListStudent_success() {
+        Student studentToDelete = model.getFilteredStudentList().get(INDEX_FIRST_PERSON.getZeroBased());
+        DeleteCommand deleteCommandStudent = new DeleteCommand(INDEX_FIRST_PERSON, PersonType.STUDENT);
+        String expectedMessageStudent = String.format(DeleteCommand.MESSAGE_DELETE_STUDENT_SUCCESS, studentToDelete);
+        ModelManager expectedModelStudent = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModelStudent.deleteStudent(studentToDelete);
+
         assertCommandSuccess(deleteCommandStudent, model, expectedMessageStudent, expectedModelStudent);
     }
 
     @Test
-    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
+    public void execute_invalidIndexUnfilteredListTutor_throwsCommandException() {
         Index outOfBoundIndexTutor = Index.fromOneBased(model.getFilteredTutorList().size() + 1);
-        Index outOfBoundIndexStudent = Index.fromOneBased(model.getFilteredStudentList().size() + 1);
-
         DeleteCommand deleteCommandTutor = new DeleteCommand(outOfBoundIndexTutor, PersonType.TUTOR);
-        DeleteCommand deleteCommandStudent = new DeleteCommand(outOfBoundIndexStudent, PersonType.STUDENT);
 
-        // Fail at deleting a tutor for invalid index for an unfiltered list
         assertCommandFailure(deleteCommandTutor, model, Messages.MESSAGE_INVALID_TUTOR_DISPLAYED_INDEX);
-        // Fail at deleting a student for invalid index for an unfiltered list
+    }
+
+    @Test
+    public void execute_invalidIndexUnfilteredListStudent_throwsCommandException() {
+        Index outOfBoundIndexStudent = Index.fromOneBased(model.getFilteredStudentList().size() + 1);
```

Comment:
You can make use of the `INVALID_INDEX` in the CommandTestUtil.java file for this.

Reply:
`INVALID_INDEX` is a String, but we want to use an Index type here, so I don't think I will change this